tracing-appender/src/rolling.rs

```diff
@@ -74,23 +74,22 @@ impl RollingFileAppender {
     /// ```rust
     /// # fn docs() {
     /// use tracing_appender::rolling::{RollingFileAppender, Rotation};
-    /// let file_appender = RollingFileAppender::new(Rotation::HOURLY, "/some/directory", "prefix.log");
+    /// let file_appender = RollingFileAppender::new(Rotation::HOURLY, "/some/directory", "prefix.log").unwrap();
```

Comment:
It might be worth adding something to the body text of the doc comment saying that this returns an error if the directory/file could not be created.

Reply:
+1. Another option would be to place the message in the `.expect()`—I think it's a good idea to show good documentation hygiene.